library/src/test/java/io/constructor/core/ConstructorIoIntegrationTest.kt

```diff
@@ -63,10 +67,43 @@ class ConstructorIoIntegrationTest {
         Thread.sleep(timeBetweenTests)
     }
 
+    @Test
+    fun getAutocompleteResultsCRTAgainstRealResponse() {
+        runBlocking {
+            launch {
+                try {
+                    val autocompleteResults = constructorIo.getAutocompleteResultsCRT("pork")
+                    assertTrue(autocompleteResults.sections!!.isNotEmpty())
+                    assertTrue(autocompleteResults.resultId!!.isNotEmpty())
+                } catch (e: Exception) {
+                    assertNull(e)
+                }
```

Comment:
I'm not exactly sure how coroutines work, but providing they don't swallow exceptions I would expect this test to work the same with and without the `try-catch` part.
If the code happens to produce an exception `assertNull` would fail and the test would fail.
If we don't catch the exception the test would still fail since we produced an exception :confused: 
Am I missing something?

Reply:
Good call, that makes sense. Will update. 👍

Reply:
Hope you tested this, I didn't :smile: